fe/fe-core/src/main/java/com/starrocks/qe/GlobalVariable.java

```diff
@@ -87,6 +96,124 @@ public final class GlobalVariable {
     @VariableMgr.VarAttr(name = "performance_schema", flag = VariableMgr.READ_ONLY)
     private static boolean performanceSchema = false;
 
+    /**
+     * Query will be pending when BE is overloaded, if `queryQueueEnable` is true.
+     * <p>
+     * If the number of running queries of any BE `exceeds queryQueueConcurrencyHardLimit`,
```

Comment:
using the drivers number may be better? a complex query may have throunds of drivers.

Reply:
No, the most users don't know what is driver and how many drivers a query contains.